js/components/addEditBookmark.js

```diff
@@ -24,9 +24,15 @@ class AddEditBookmark extends ImmutableComponent {
     this.onSave = this.onSave.bind(this)
     this.onRemoveBookmark = this.onRemoveBookmark.bind(this)
   }
+
   get isBlankTab () {
     return ['about:blank', 'about:newtab'].includes(this.props.currentDetail.get('location'))
   }
+
+  get bookmarkNameValid () {
+    return (this.props.currentDetail.get('title') || this.props.currentDetail.get('customTitle'))
```

Comment:
Actually in case of Bookmark `this.props.currentDetail.get('title')` will always be set as the website's title, so even if you have empty customTItle, you should not see grayed button and if you submit the title will fallback to website's title.